packages/server/tests/AuthenticationSignUp.test.ts

```diff
@@ -66,6 +68,17 @@ test.group('Authentication sign up tests group', (group) => {
                 message.subject,
                 `Welcome ${userNickname}, please verify your email !`,
             );
+
+            const html = message.html;
+            invariant(
+                html !== undefined,
+                'HTML content of the email must be defined',
+            );
+
+            const $ = cheerio.load(html);
```

Comment:
Question: You don't use adonis/view to avoid any black box testing issue ?

Reply:
As we discussed, I don't think @adonisjs/view can parse HTML. Although it's the job of tools such as cheerio.